src/GovUk.Education.ExploreEducationStatistics.Content.ViewModels/GovUk.Education.ExploreEducationStatistics.Content.ViewModels.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\GovUk.Education.ExploreEducationStatistics.Data.Services\GovUk.Education.ExploreEducationStatistics.Data.Services.csproj" />
```

Comment:
Shouldn't need this dependency. Think this can be replaced with a reference to `Content.Model` instead

Reply:
Refactored so that this no longer depends on `Data.Services`. I've pulled the view models out of `Data.Services` into their own project `Data.ViewModels`